doc/specs/stdlib_linalg.md

```diff
@@ -6,6 +6,62 @@ title: linalg
 
 [TOC]
 
+The `stdlib` linear algebra library provides high-level APIs for dealing with common linear algebra operations.
+
+## BLAS and LAPACK
+
+### Status
+
+Experimental
+
+### Description
+
+`BLAS` and `LAPACK` backends provide efficient low level implementations of many linear algebra algorithms, and are employed for non-trivial operators. 
+A Modern Fortran version of the [Reference-LAPACK 3.10.1](http://github.com/reference-LAPACK) implementation is provided as a backend. 
```

Comment:
I've noticed that other libraries use older versions, so I wanted to stay on the safe side to avoid potential issues from the newest changes. 
For example, NumPy uses 3.2: https://github.com/numpy/numpy/blob/c3694335faffe35e3d961feffefdc0a4b9a665da/numpy/linalg/lapack_lite/f2c_c_lapack.c#L76-L79
Apple uses `3.9.1`: https://developer.apple.com/documentation/accelerate/blas/

I agree it would be nice to gradually move towards the newer versions and hopefully it shouldn't be a big deal, since most of the process is automated